### PR TITLE
Us#15 dev merge

### PR DESCRIPTION
**Pull request by ConfuzzedCat**

### CODE REVIEW

File: backend/src/main/java/dk/lyngby/Model/Route.java
Comment (HeinLarsen) [inline backend/src/main/java/dk/lyngby/Model/Route.java:1]: This file is not needed. Use model/Route (small m)

File: backend/src/main/java/dk/lyngby/model/Route.java [hunk 1]
```diff
@@ -1,14 +1,18 @@
 package dk.lyngby.model;
 import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonFormat;
```
Comment (HeinLarsen) [inline backend/src/main/java/dk/lyngby/model/Route.java]: duplicate imports?

File: frontend/src/rides.jsx [hunk 2]
```diff
@@ -0,0 +1,155 @@
+import React, { useState, useEffect } from "react";
+
+function RideModal({ ride, onClose }) {
+    if (!ride) return null;
+
+    // Definer styles osv, for at undgå at bruge App.css fordi der blev snakket om den skulle slettes
+    const modalStyle = {
```
Comment (HeinLarsen) [inline frontend/src/rides.jsx:7]: This could be done with tailwind classes instead. No need to update it now just fyi for the future

**Review by HeinLarsen — CHANGES_REQUESTED** [review]

File: frontend/src/rides.jsx (hunk 2 above)
Comment (HeinLarsen) [reply frontend/src/rides.jsx:7]: Seems like github didn't like me marking multiple lines. I meant the CSS classes in rides.jsx

Comment (ConfuzzedCat) [reply frontend/src/rides.jsx:7]: @roastedtoastyt

**Review by HeinLarsen — APPROVED** [review]

**Pull request merged**